src/main/java/securbank/controller/ManagerController.java

```diff
@@ -41,6 +51,90 @@ public String currentUserDetails(Model model) {
         return "manager/detail";
     }
 	
+	@GetMapping("/manager/edit")
+    public String editUser(Model model) {
+		User user = userService.getCurrentUser();
+		if (user == null) {
+			return "redirect:/error";
+		}
+		model.addAttribute("user", user);
+		logger.info("GET request: Manager profile edit");
+		
+        return "manager/edit";
+    }
+	
+	@PostMapping("/manager/edit")
+    public String editSubmit(@ModelAttribute ModificationRequest request, BindingResult bindingResult) {
+		editUserFormValidator.validate(request, bindingResult);
+		if (bindingResult.hasErrors()) {
+			return "manager/edit";
+        }
+		
+		// create request
+    	userService.createInternalModificationRequest(request);
+    	logger.info("POST request: Manager New modification request");
+    	
+        return "redirect:/";
+    }
+	
+	@GetMapping("/manager/user/request")
+    public String getAllUserRequest(Model model) {
+		List<ModificationRequest> modificationRequests = userService.getModificationRequests("pending", "external");
+		if (modificationRequests == null) {
+			model.addAttribute("modificationrequests", new ArrayList<ModificationRequest>());
+		}
+		else {
+			model.addAttribute("modificationrequests", modificationRequests);	
+		}
+		logger.info("GET request: Manager All external modification requests");
+		
+        return "manager/modificationrequests";
+    }
+	
+	@GetMapping("/manager/user/request/{id}")
+    public String getUserRequest(Model model, @PathVariable() UUID id) {
+		ModificationRequest modificationRequest = userService.getModificationRequest(id);
+		
+		if (modificationRequest == null) {
+			return "redirect:/error?=code=400&path=request-invalid";
+		}
+		
+		model.addAttribute("modificationrequest", modificationRequest);
+		logger.info("GET request: Manager external modification request by ID");
+		
+        return "manager/modificationrequest_detail";
+    }
+	
+	@PostMapping("/manager/user/request/{requestId}")
+    public String approveEdit(@PathVariable UUID requestId, @ModelAttribute ModificationRequest request, BindingResult bindingResult) {
+		String status = request.getStatus();
+		if (status == null || !(request.getStatus().equals("approved") || !request.getStatus().equals("rejected"))) {
```

Comment:
You've fixed the bug in all the other files but I think you missed it over here.